docs/schema/condition-element-workflowinfo.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Condition element (WorkflowInfo)
+description: A Condition element represents a Condition statement, which is part of a rule sentence that can be displayed in a declarative, rules-based, code-free workflow editor.
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.